user_sync/app.py

```diff
@@ -105,11 +105,16 @@ def init_log(logging_config):
     }
     
     console_log_level = level_lookup.get(options['console_log_level'])
-    if (console_log_level != None):
-        console_log_handler.setLevel(console_log_level)
-    
+    if (console_log_level == None):
+        console_log_level = logging.DEBUG
+        logger.log(logging.INFO, 'Unknown console log level: %s setting to debug' % options['console_log_level'])
+    console_log_handler.setLevel(console_log_level)
+
     if options['log_to_file'] == True:
-        file_log_level = level_lookup.get(options['file_log_level'], logging.NOTSET)
+        file_log_level = level_lookup.get(options['file_log_level'])
+        if (file_log_level == None):
+            file_log_level = logging.DEBUG
+            logger.log(logging.INFO, 'Unknown file log level: %s setting to debug' % options['file_log_level'])
```

Comment:
This line (and the corresponding one above seem problematic.  Either we are going to be emitting this message about unknown log level even if no log level has been specified, or this code is not running to set the default if there is no specification in the file.  Seems like this line (and the one above) should only emit if there is an invalid value specified.

Also, this should be a warning, not info.